includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -136,7 +136,17 @@ public function process() {
 	 * @param array $atts Attributes to use in the view handler.
 	 */
 	public function output( $atts = [] ) {
-		$this->enqueue_scripts();
+		$resumes_enabled_recaptcha = (bool) get_option( 'resume_manager_enable_recaptcha_resume_submission', false );
+		if ( is_array( $atts ) ) {
+			if ( in_array( 'resume', $atts, true ) ) {
+				if ( $resumes_enabled_recaptcha ) {
+					WP_Job_Manager\WP_Job_Manager_Recaptcha::enqueue_scripts();
+				}
+			} else {
+				WP_Job_Manager\WP_Job_Manager_Recaptcha::enqueue_scripts();
+			}
+		}
```

Comment:
This can be simplified with:
```suggestion
		if ( is_array( $atts ) ) {
			if ( ! in_array( 'resume', $atts, true ) || $resumes_enabled_recaptcha ) {
				WP_Job_Manager\WP_Job_Manager_Recaptcha::enqueue_scripts();
			}
		}
```

Reply:
Good call!  d6cb78736bef930b891d5d26b7576b1a6dc623f8